index.md

```diff
@@ -3,72 +3,134 @@ title: Amazon Ion
 ---
 
-Amazon Ion is a [richly-typed][13], [self-describing][15], hierarchical data serialization
+<div style="float:right; width:200px" markdown="block">
+
+| [News][7] |
+|------|{% for post in site.posts limit:5 %}
+|<a href="{{site.baseurl}}{{post.url}}">{{ post.title }}</a>|{% endfor %}
+
+</div>
+
+**<font size="+1">Amazon Ion</font>** is a [richly-typed][13], [self-describing][15], hierarchical data serialization
 format offering [interchangeable binary and text][14] representations. The [text format][10]
 (a superset of [JSON][1]) is easy to read and author, supporting rapid
 prototyping. The [binary representation][11] is [efficient to store, transmit, and
 skip-scan parse][16].  The rich type system provides unambiguous semantics for
-long-term preservation of business data which can survive multiple generations
+long-term preservation of data which can survive multiple generations
 of software evolution.
 
-**Available Libraries:** [Ion Java][3] -- [Ion C][4] -- [Ion Python][5] -- [Ion JavaScript][6]
+Ion was built to address rapid development, decoupling, and efficiency
+challenges faced every day while engineering large-scale, service-oriented
+architectures. It has been addressing these challenges within Amazon for nearly
+a decade, and we believe others will benefit as well.
+
+**Available Libraries:** [Ion Java][3] -- [Ion C][4] -- [Ion Python][5] -- [Ion JavaScript][6]<br>
+**Related Projects:** [Ion Schema][17]<br>
+**Tools:** [Hive SerDe][18]<br>
 
-**Related Projects:** [Ion Schema][17]
+<br>
 
-**Tools:** [Hive SerDe][18]
+### Getting Started {#gettingstarted}
 
-<br/>
+All JSON values are valid Ion values, and while any value can be encoded in JSON (e.g., a timestamp value can be converted to a string), such approaches require extra effort, obscure the actual type of the data, and tend to be error-prone.
 
-### Latest News
+In contrast, Ion's rich type system enables unambiguous semantics for data (e.g., a timestamp value can be encoded using the timestamp type).  The following illustrates some of the features of the Ion type system:
 
----
-{% for post in site.posts limit:2 %}
-  **<a href="{{site.baseurl}}{{post.url}}">{{ post.title }}</a>**<br/>
-  *{{post.date | date_to_long_string}}*<br/>
-  {{post.content}}
-{% endfor %}
----
-Visit the [News][7] page for more announcements about Amazon Ion.
+* **timestamp:**  arbitrary precision date / timestamps
+```
+2003-12-01T
+2010-03-22T18:00:00Z
+2019-05-01T18:12:53.472-0800
+```
+
+* **int:**  arbitrary size integers
+```
+0
+-1
+12345678901234567890...
+```
+
+* **decimal:**  arbitrary precision, base-10 encoded real numbers
+```
+0.
+-1.2
+3.141592653589793238...
+6.62607015d-34
+```
+
+* **float:**  32-/64-bit IEEE-754 floating-point values
+```
+0e0
+-1.2e0
+6.02e23
+-inf
+```
+
+* **symbol:**  provides efficient encoding for frequently occurring strings
+```
+inches
+dollars
+'high-priority'    // symbols with special characters are enclosed in single-quotes
```

Comment:
I would add in the comment that the special character here is `-`

Reply:
added comment re `-`